apps/chat-e2e/src/ui/webElements/sideBar.ts

```diff
@@ -148,6 +148,28 @@ export class SideBar extends BaseElement {
     await this.page.mouse.up();
   }
 
+  public async dragAndDropEntityToRoot(
```

Comment:
seems this method is identical to `dragAndDropFolderToRoot()`

Reply:
now it is different and one calls another